libctru/include/3ds/gpu/gx.h

```diff
@@ -2,6 +2,29 @@
 
 #define GX_BUFFER_DIM(w, h) (((h)<<16)|((w)&0xFFFF))
 
+typedef enum
+{
+	GX_TRANSFER_FMT_RGBA8  = 0,
+	GX_TRANSFER_FMT_RGB8   = 1,
+	GX_TRANSFER_FMT_RGB565 = 2,
+	GX_TRANSFER_FMT_RGB5A1 = 3,
+	GX_TRANSFER_FMT_RGBA4  = 4
+} GX_TRANSFER_FORMAT;
+
+typedef enum
+{
+	GX_TRANSFER_SCALE_NO = 0,
+	GX_TRANSFER_SCALE_X  = 1,
+	GX_TRANSFER_SCALE_Y  = 2
```

Comment:
doesn't this flag scale down both X and Y ?
also the flags after this line lack the truncate bit, but that can be added later I guess.